kbi/0006/index.rst

```diff
@@ -0,0 +1,70 @@
+.. index::
+   single: <topic>; <subtopic>
+
+KBI0006: How to fix-up a git-type special remote with a new location
+====================================================================
+
+:authors: Adina Wagner <adina.wagner@t-online.de>
+:discussion: <link>
+:keywords: special remote, git-annex, gitannex; enableremote
+
+In many datasets, special remotes ensure that annexed file content can
+be retrieved from external sources.
+Sometimes, however, external resources vanish.
+This happened to data of the studyforrest project:
+Initially, data was hosted on University of Magdeburg's
+infrastructure (``psydata.ovgu.de``), and registered as a `Git-type special remote`_
+``mddatasrc``:
+
+.. code-block:: bash
```

Comment:
```suggestion
.. code-block:: console
```

I recently learned that Pygments has a [BashSessionLexer](https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer) for "Bash shell sessions, i.e. command lines, including a prompt, interspersed with output" (short names: `console`, `shell-session`). For me it colored the prompt and greyed out the output:

![Screenshot from 2023-04-17 22-00-35](https://user-images.githubusercontent.com/11985212/232597814-cca1b5a1-fa34-4052-8ac6-6bcfe990ef35.png)

If you like it, you can also set it in code blocks below / per document. Or reject the suggestion.

Reply:
oh, this is cool! How could I "set it per document"?

Reply:
`.. highlight:: console` near the beginning should work "on a document-wide basis"

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
oh awesome, thanks for sharing!

Reply:
tried it in [c30e8a5](https://github.com/psychoinformatics-de/knowledge-base/pull/22/commits/c30e8a55d1e11dc653d63557bdf8dec298f5236f)